datanommer.models/alembic/versions/1d4feffd78fe_add_historic_user_an.py

```diff
@@ -83,15 +83,15 @@ def upgrade():
 
         if not msg.users:
             new_usernames = msg2usernames(msg.__json__(), **config)
-            print "Updating users to %r" % new_usernames
+            print("Updating users to %r" % new_usernames)
```

Comment:
That comment is wrong; `("Updating users to %r" % new_usernames)` is *not* a tuple with one element.

`(value)` is just the value; a one-element tuple would be spelled `(value, )` – note the comma.